envoy/README.md

```diff
@@ -155,16 +158,38 @@ If you care only about the cluster name and grpc service, you would add this to
 
 3. [Restart the Agent][9].
 
+#### Containerized
+
+For containerized environments, see the [Autodiscovery Integration Templates][11] for guidance on applying the parameters below.
+
+##### Metric collection
```

Comment:
It would have been nice if we could provide an example AD integration template for each integration. Otherwise users who are not familiar with AD would need to navigate to multiple places and may make mistakes before getting it right

Reply:
This is planned for future updates. The "recipes" are the 1st step.